drivers/FGD-212/device.js

```
@@ -37,33 +37,39 @@ class FibaroDimmerTwoDevice extends ZwaveDevice {
 				};
 
 				switch (this.getSetting('switch_type')) {
-					case '0': this._momentaryTrigger.trigger(this, null, data); break;
-					case '1': this._toggleTrigger.trigger(this, null, data); break;
-					case '2': this._rollerTrigger.trigger(this, null, data); break;
+				case '0':
+					this._momentaryTrigger.trigger(this, null, data);
+					break;
+				case '1':
+					this._toggleTrigger.trigger(this, null, data);
+					break;
+				case '2':
+					this._rollerTrigger.trigger(this, null, data);
+					break;
 				}
 			}
 		});
 	}
 
 	async _setBrightnessRunListener(args, state) {
-		if (!args.hasOwnProperty('set_forced_brightness_level')) return Promise.reject('set_forced_brightness_level_property_missing');
-		if (typeof args.set_forced_brightness_level !== 'number') return Promise.reject('forced_brightness_level_is_not_a_number');
-		if (args.set_forced_brightness_level > 1) return Promise.reject('forced_brightness_level_out_of_range');
-
-		if (args.hasOwnProperty('set_forced_brightness_level') && typeof args.set_forced_brightness_level === 'number') {
-			const value = Math.round(args.set_forced_brightness_level * 99);
-
-			if (this.node.CommandClass.COMMAND_CLASS_CONFIGURATION) {
-				return this.node.CommandClass.COMMAND_CLASS_CONFIGURATION.CONFIGURATION_SET({
-					'Parameter Number': 19,
-					Level: {
-						Size: 1,
-						Default: false,
-					},
-					'Configuration Value': new Buffer([value]),
-				});
-			} return Promise.reject('configuration_command_unavailable');
-		} return Promise.reject('unknown_error');
+		if (!args.hasOwnProperty('set_forced_brightness_level')) throw Error('set_forced_brightness_level_property_missing');
```

Is there a reason behind `throw Error` instead of return `Promise.reject()`?

---

I'll revert van to `return Promise.reject()`